table/index.go

```diff
@@ -27,12 +27,18 @@ type IndexIterator interface {
 	Close()
 }
 
+// CreateIdxOpt contains the options will be used when creating an index.
+type CreateIdxOpt struct {
+	SkipHandleCheck bool // If true, skip the handle constraint check.
+	SkipCheck       bool // If true, skip all the unique indices constraint check.
```

Comment:
SkipCheck is useless?

Reply:
it will be used in following complex PR, this one just a refactor